docs-client/package-lock.json

```diff
@@ -3369,6 +3369,16 @@
       "dev": true,
       "optional": true
     },
+    "bindings": {
```

Comment:
Is this change relevant to this pull request? :thinking:

Reply:
Gradle build just keeps changing this file.
I made a separate PR to solve this. https://github.com/line/armeria/pull/2777